manager/api/requests.go

```diff
@@ -110,16 +110,16 @@ func (req viewResourceReq) validate() error {
 
 type listResourcesReq struct {
 	key    string
-	size   int
 	offset int
+	limit  int
 }
 
 func (req listResourcesReq) validate() error {
 	if req.key == "" {
 		return manager.ErrUnauthorizedAccess
 	}
 
-	if req.size > 0 && req.offset >= 0 {
+	if req.offset >= 0 && req.limit > 0 {
```

Comment:
Reading the docs pointed me to another issue: `limit` cannot be infinite. Please set the upper bound and ensure the API documentation is synchronized.

P.S.: don't forget the tests.